docker/dev.Dockerfile

```diff
@@ -27,6 +26,5 @@ ADD . /app
 WORKDIR /app
 
 # Common and dev deps installed separately to prove that common.txt works standalone
```

Comment:
I don't quite follow why this is needed... is it only because of how heroku installed requirements?

Reply:
You may already figured this out, but I _think_ what this comment is explaining is that installing `common.txt` in its own call to `pip install` will make sure it always works, and that it doesn't depend on `dev.txt` to succeed. If `common.txt` was missing a dependency that `dev.txt` provided, the install would fail otherwise, I believe.